### PR TITLE
Export endpoint

### LINKED ISSUE #474: Login endpoint for external tournament planning system
Ask Paul for the details, I don't know the exact specifics, but with this, we don't have to make a database export before the event.

### PR DESCRIPTION
**Pull request by skamoen**
To make exporting and importing data easier for Paul, this adds an export endpoint containing the most essential fields about users and teams

Fixes #474

### CODE REVIEW

File: src/main/java/ch/wisv/areafiftylan/seats/model/Seat.java [hunk 1]
```diff
@@ -65,4 +65,9 @@ public User getUser() {
     public boolean isTaken() {
         return ticket != null;
     }
+
+    @Override
+    public String toString() {
+        return "" + seatGroup + seatNumber;
```
Comment (martijnjanssen) [inline src/main/java/ch/wisv/areafiftylan/seats/model/Seat.java]: This looks hideous, please change this.

File: src/main/java/ch/wisv/areafiftylan/users/model/UserExportDTO.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+public class UserExportDTO {
+
+    private String email;
+    private String password_hash;
+    private Long userId;
+    private List<Seat> seat;
```
Comment (martijnjanssen) [inline src/main/java/ch/wisv/areafiftylan/users/model/UserExportDTO.java]: Why not remove this here and add it as a separate export? An export to a `SeatExportDTO` with `Long userId` and `String seatString`?

File: src/main/java/ch/wisv/areafiftylan/utils/ExportController.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    private final TicketService ticketService;
+
+    public ExportController(SeatService seatService, TeamService teamService, TicketService ticketService) {
+        this.seatService = seatService;
+        this.teamService = teamService;
+        this.ticketService = ticketService;
+    }
+
+    @GetMapping
+    public Map<String, ?> exportUsers() {
```
Comment (martijnjanssen) [inline src/main/java/ch/wisv/areafiftylan/utils/ExportController.java]: This function looks horribly formatted here, that needs to improve.

File: src/main/java/ch/wisv/areafiftylan/users/model/UserExportDTO.java (hunk 2 above)
Comment (skamoen) [reply src/main/java/ch/wisv/areafiftylan/users/model/UserExportDTO.java]: A User has a seat, why would you export that separately. It's all tied to the userId, which is the whole point of this endpoint right?

Comment (martijnjanssen) [reply src/main/java/ch/wisv/areafiftylan/users/model/UserExportDTO.java]: True, but if we want to export more, it's more expandable. It looks a little sloppy and more thrown together now. We could use it this way, but it looks a bit low quality :stuck_out_tongue:

**Pull request merged**